extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -18,12 +20,26 @@ export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProj
 export const UpdateNetCoreLocation: string = localize('sqlDatabaseProjects.UpdateNetCoreLocation', "Update .Net Core location");
 export const InstallNetCore: string = localize('sqlDatabaseProjects.InstallNetCore', "Install .Net Core SDK");
 
+const projectsOutputChannel = localize('sqlDatabaseProjects.outputChannel', "Database Projects");
+const dotnet = os.platform() === 'win32' ? 'dotnet.exe' : 'dotnet';
+
+export interface DotNetCommandOptions {
+	workingDirectory?: string;
+	additionalEnvironmentVariables?: NodeJS.ProcessEnv;
+	commandTitle?: string;
+	argument?: string;
```

Comment:
Just double-checking: none of these options are required?

Reply:
yeah not really